jenkins/jobs/ci/tidb/ghpr_integration_br_test.groovy

```diff
@@ -24,9 +24,13 @@ pipelineJob('tidb_ghpr_integration_br_test') {
                     whitelist('')
                     orgslist('pingcap')
                     whiteListTargetBranches {
-                        ghprbBranch { branch('master') }
                         ghprbBranch { branch('^(release-)?5\\.[0-4]\\d*(\\.\\d+)?(\\-.*)?$') }
                         ghprbBranch { branch('^(release-)?6\\.[0-9]\\d*(\\.\\d+)?(\\-.*)?$') }
+                    }
+                    blackListTargetBranches {
```

Comment:
Do we need both blacklist and whitelist?

Reply:
Yes, currently, the reconstructed pipeline only supports the master and lst branches. There may be a need to trigger validation for the dmr branch.

Reply:
However, the dmr branch will not continue to merge PRs. How about we directly disable triggering for all branches?